src/lib/invite.js

```diff
@@ -38,22 +38,14 @@ const TRANSACTION_STATES = {
     label: 'Signing Transactions...',
     progress: 0.15,
   },
-  FUNDING_INVITE: {
-    label: 'Funding Invite Wallet...',
+  FUNDING: {
+    label: 'Funding Transactions...',
     progress: 0.3,
   },
   CLAIMING: {
     label: 'Claiming Invite...',
     progress: 0.55,
```

Comment:
perhaps we should update the progress percent here, since there are fewer steps?

Reply:
I did actually change these values around a bit to try and fat-finger estimate the time they would take, but ended up at the same values as they initially had. Reasoning here for funding taking 25% while claiming takes 40% is that funding is _at most_ a small ETH transfer transaction with high gas price, so should be included fairly quickly. The same cannot be said for multiple larger (but still high gas price) transactions.

Reply:
sgtm!